lib/fastlane/actions/update_info_plist.rb

```diff
@@ -0,0 +1,85 @@
+module Fastlane
+  module Actions
+    module SharedValues
+    end
+
+    class UpdateInfoPlistAction < Action
+      def self.run(params)
+        require 'plist'
+                
+        # Check if parameters are set
+        if params[:app_identifier] or params[:display_name]
+          
+          # Assign folder from parameter or search for xcodeproj file
+          folder = params[:xcodeproj] || Dir["*.xcodeproj"].first
+        
+          # Read existing plist file 
+          info_plist_path = File.join(folder, "..", params[:plist_path])
+          raise "Couldn't find info plist file at path '#{params[:plist_path]}'".red unless File.exists?(info_plist_path)
```

Comment:
As far as I understand the user has to provide both a path to a project and a path to the plist. Why not do that in one parameter? 

If we leave it like this, we'd still have to change the output probably:

from
```ruby
raise "Couldn't find info plist file at path '#{params[:plist_path]}'".red unless File.exists?(info_plist_path)
```
to
```ruby
raise "Couldn't find info plist file at path '#{info_plist_path}'".red unless File.exists?(info_plist_path)
```